.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,12 @@
+name: EasyBuild framework unit tests
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
```

Comment:
Hmm, this doesn't seem to be working as expected... The `python` command that `eb` picks up on is `2.7.15+` for all 4 of these environment (and `eb` considers using `python3` before `python`).

Reply:
fixed in c067736